index.html

```diff
@@ -320,18 +320,17 @@
           view in degrees.
         </p>
         <p>
-          A <a>depth map</a> has an associated <dfn>unit</dfn> which is a
-          string. It represents the <a>active depth map unit</a>.
+          A <a>depth map</a> has an associated <dfn>precision</dfn> which is a
+          string. It represents the <a>depth precision</a> (one of the
+          <a>possible depth precisions</a>).
         </p>
         <p>
           A <a>depth map</a> has an associated <dfn>near value</dfn> which is a
-          double. It represents the minimum range in <a>active depth map
-          unit</a>s.
+          double. It represents the minimum range in millimeters.
```

Comment:
I'd prefer the `depthNear` and `depthFar` follow `depthMapUnit`. And as today's hardware commonly produce 16bit depth value in millimeter, so I prefer the default unit is `mm`.